examples/asg_elb/main.tf

```diff
@@ -101,7 +101,8 @@ module "example_asg" {
 # ELB
 ######
 module "elb" {
-  source = "terraform-aws-modules/elb/aws"
+  source  = "terraform-aws-modules/elb/aws"
+  version = "2.4.0"
```

Comment:
Pinning the ELB module to 2.4.0 to work around the provider source warning in the latest release causing the checks to fail.